src/math/utils/quantizeForSpace.js

```diff
@@ -0,0 +1,6 @@
+const { spatialResolution } = require('../constants')
+
+ // Quantize values for use in spatial coordinates, and so on.
+const quantizeForSpace = (value) => (Math.round(value * spatialResolution) / spatialResolution)
```

Comment:
Wow. I looked up the definition of quantize... I don’t think this has anything to do with quants. How about roundToSpaceResolution(). Now, I know exactly what’s happening. 😃

Reply:
Not sure what quants have to do with it. :)

The definition I've been working with is:

quan·tize
verb
1.
PHYSICS
apply quantum theory to, especially form into quanta, in particular restrict the number of possible values of (a quantity) or states of (a system) so that certain variables can assume only certain discrete magnitudes.
"light is quantized into packets of energy"
2.
ELECTRONICS
approximate (a continuously varying signal) by one whose amplitude is restricted to a prescribed set of values.
"distortion is caused when very low-level audio signals are quantized"

Let me know if that works